docs/about/pricing.md

```
@@ -22,6 +21,18 @@ CodeRabbit.
 
 ---
 
+## Open Source
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)